pages/wishlist/index.html

```diff
@@ -0,0 +1,198 @@
+                src="/media/icons/heartFilled.svg"
+                alt="heart icon"
+                class="nav-icon"
+              />
+              Wishlist
+            </a>
+          </li>
+          <li>
+            <a class="nav-link" href="/pages/cart/"
+              ><img
+                src="/media/icons/cart.svg"
+                alt="cart icon"
+                class="nav-icon"
+              />
+              Cart</a
+            >
+          </li>
+        </ul>
+        <!-- end of nav-links -->
+      </nav>
+      <!-- end of nav-head -->
+    </header>
+
+    <!-- showcase for heading -->
+    <div class="parting flex-row">
+      <a href="#" class="text-dec-none text-link med-text">My Wishlist</a>
+    </div>
+    <!-- end of showcase -->
+
+    <!-- start of main content -->
+    <main class="flex-center gap-xl mg-m">
+      <!-- card-1 -->
+      <div class="card-ecom">
+        <button class="like-btn">
+          <img src="/media/icons/redHeart.svg" alt="heart icon" />
+        </button>
+        <img
+          src="/media/images/stackedSoap.webp"
+          alt="handmade soap set"
+          class="responsive-img"
+        />
+        <div class="product-details flex-center flex-col gap-sm">
+          <h4 class="marg-un">Vinaya Beauty</h4>
+          <p class="marg-un">Handmade soap set</p>
+          <div><strong> Rs. 749 </strong> <s>Rs.999</s></div>
+        </div>
+        <button class="cart-btn gap-sm">Move to cart</button>
+      </div>
+      <!-- card 2 -->
+      <div class="card-ecom">
+        <button class="like-btn">
+          <img src="/media/icons/redHeart.svg" alt="heart icon" />
+        </button>
+        <img
+          src="/media/images/basket1.webp"
+          alt="handmade woven storage basket"
+          class="responsive-img"
+        />
+        <div class="product-details flex-center flex-col gap-sm">
+          <h4 class="marg-un">Nature jewels</h4>
+          <p class="marg-un">Storage basket</p>
+          <div><strong> Rs. 1299 </strong> <s>1699</s></div>
+        </div>
+        <button class="cart-btn gap-sm">Move to cart</button>
+      </div>
+    </main>
+    <!-- end of main content -->
+
+    <footer>
+      <div class="footer-grid">
+        <!-- icon-links -->
+        <div class="display-block">
+          Keep in touch
+          <ul class="footer-list">
+            <li>
+              <a
+                href="https://www.linkedin.com/in/surbhi-kukreti-a91b0b163"
+                target="blank"
+                class="footer-icon"
+              >
+                <i class="fab fa-linkedin-in"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/surbhi84"
+                target="blank"
+                class="footer-icon"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://twitter.com/Surbhikukreti89"
```

Comment:
why so much space used

Reply:
this space is added by the linting library it increases the readability of the code.